src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.AcceptAllCerts.cs

```diff
@@ -38,16 +39,21 @@ public void SingletonReturnsTrue()
         [InlineData(SslProtocols.None, true)]
         public async Task SetDelegate_ConnectionSucceeds(SslProtocols acceptedProtocol, bool requestOnlyThisProtocol)
         {
+            if (PlatformDetection.IsMacOsHighSierraOrHigher && acceptedProtocol == SslProtocols.Tls)
```

Comment:
Should we do it only on Catalina?  This test works fine up to 10.14. Also it fails only with PlatformHttpHandler. SocketHttpHandler handler is fine when SslStream is on both ends e.g. OSX is compatible with it self.

Reply:
Good point about SocketHttpHandler. I feel `10.14 <= version < Catalin` was only passing because of overriding `requestOnlyThisProtocol` but will relax it a bit so we run it on more OSXes.